contrib/aerc/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "aerc"
+pkgver = "0.15.2"
+pkgrel = 0
+build_style = "makefile"
+make_check_target = "tests"
+hostmakedepends = [
+    "scdoc",
+]
+makedepends = [
+    "go",
+    "notmuch-devel",
+]
+checkdepends = [
+    "gnupg",
+    "gpgme",
+]
+go_build_tags = ["notmuch"]
```

Comment:
this probably doesn't do anything

Reply:
the script it sources by default checks if [this thing](https://git.sr.ht/~rjarry/aerc/tree/master/item/contrib/goflags.sh) compiles, then passes it. passing it like this forces it so it doesn't get silently disabled if that fails for some reason

wdyt?

Reply:
ah, though it's ignored for that anyway, sure

Reply:
not sure what you mean, but the tags only get passed when using go build via the util module, which does not happen here (as it uses a makefile)

Reply:
yeah, i assumed the tags were always added to GOFLAGS (`GOFLAGS=-tags $go_build_tags`). maybe they should? sometimes that would be useful to put `libsqlite3` in there even when not using go buildstyle